src/Controls/StreetControls.js

```diff
@@ -77,8 +160,39 @@ class StreetControls extends FirstPersonControls {
         this.previousPosition = newPreviousPosition;
     }
 
+    onMouseUp(event) {
+        super.onMouseUp();
+        if (this._stateOnMouseDrag) {
+            this._stateOnMouseDrag = false;
+        } else {
+            pick(event, this.view, this.buildingsLayer, this.onClickOnGround.bind(this), this.onClickOnWall.bind(this));
+        }
+    }
+
+    onMouseMove(event) {
```

Comment:
I have an error when moving the mouse before the layer is ready. Could you add a condition like `if (!layer.ready)` ? Or something that fixes the error.

Reply:
Yes, thanks, I 've added a bugfix in View